typings/axe-core/axe-core-tests.ts

```diff
@@ -1,7 +1,7 @@
 import * as axe from '../../axe';
 
 var context: any = document;
-var $fixture: any = {};
+var $fixture: NodeList = document.querySelectorAll('div');
```

Comment:
`NodeList` is not quite right here. QSA returns `NodeList<HTMLtypeElement>`, where `type` is `Div` in this case, so this should be:

```suggestion
var $fixture: NodeList<HTMLDivElement> = document.querySelectorAll('div');
```

However, declaring any type here is redundant, as the compiler knows what QSA returns. IMO this should just be:

```suggestion
var $fixture = document.querySelectorAll('div');
```